docs/reference/cluster.asciidoc

```diff
@@ -6,23 +6,53 @@
 ["float",id="cluster-nodes"]
 == Node specification
 
-Most cluster level APIs allow to specify which nodes to execute on (for
-example, getting the node stats for a node). Nodes can be identified in
-the APIs either using their internal node id, the node name, address,
-custom attributes, or just the `_local` node receiving the request. For
-example, here are some sample executions of nodes info:
+Most cluster level APIs allow to specify nodes with `node filters`, these filters
+limit the results to selected nodes for corresponding APIs. (For example, you can
+find it in APIs like `Cluster State`, `Task Management` and all the Nodes API like
+`Nodes States`, `Nodes Info` etc.)
```

Comment:
I'm not sure that _most_ cluster-level APIs allow us to specify a set of nodes - by my reckoning it's these:

- Cluster Stats (via the (undocumented?) `?nodeId=` parameter - could also do with fixing)
- Nodes Stats
- Nodes Info
- Nodes Feature Usage
- Task Management API
- Nodes hot_threads

There are 13 subheadings in this section, and 6/13 is less than half. (I haven't dug through the code to confirm this so my numbers could be off). In any case it might not be more than half in future, so I think I'd prefer "some" to "most".

Also the things in `backticks` in this paragraph are not code so I don't think they should be formatted like this. The ones that refer to other pages would be better as links, and `node filters` is introducing a piece of terminology which we seem either to put _in italics_ or else "in double quotes" depending on taste.

Also also `Nodes States` should probably be `Nodes Stats`, and the cluster state API doesn't seem to support a node filter as stated here.

Reply:
Yes that's right, the `Cluster Stats` API doesn't support node filters, in fact we can only add `nodes` to filter the `nodes` part of the response but no further filterings.